index.html

```diff
@@ -3057,14 +3062,26 @@ <h4>
             listed above.
           </p>
           <p>
-            The <a>top-level browsing context</a> MUST NOT be allowed to
-            navigate itself, except by <a>navigating to a fragment
-            identifier</a> or by <a data-lt="reload a document">reloading its
-            document</a>. This allows the user to grant permission based on the
+            When the <a>top-level browsing context</a> attempts to navigate to
+            a new resource and runs the <a>steps to navigate</a>, it MUST
+            follow step 1 to determine if it is <a>allowed to navigate</a>. In
+            addition, it MUST NOT be allowed to navigate itself to a new
+            resource, except by <a>navigating to a fragment identifier</a> or
+            by <a data-lt="reload a document">reloading its document</a>.
+          </p>
+          <p class="note">
+            This allows the user to grant permission based on the origin of the
             presentation URL shown when <a data-lt=
             "select a presentation display">selecting a presentation
             display</a>.
           </p>
+          <p>
+            If the top-level-browsing context was not <a>allowed to
+            navigate</a>, it SHOULD NOT offer to open the resource in a new
+            <a>top-level browsing context</a>, but instead SHOULD abort the
+            navigation and possibly throw a <a>SecurityError</a> (consistent
+            with the <a>steps to navigate</a>).
```

Comment:
That is not entirely "consistent" with the steps to navigate: the steps to navigate in HTML mandate the `SecurityError` be thrown when the [exceptions enabled flag](https://www.w3.org/TR/html51/browsers.html#exceptions-enabled-flag) is set, whereas this makes it optional. 

If the `SecurityError` exception cannot be mandated because it remains a challenge implementation-wise, I would suggest to drop "(consistent with the steps to navigate)".

Reply:
Throwing the exception isn't an implementation concern from my point of view.  Rather I didn't want to get into the details here of when to throw a `SecurityError` and thus duplicate HTML.  I rephrased this, let me know if that addresses your feedback.

Reply:
Yes, looks good!